chroma-common/Makefile

```diff
@@ -0,0 +1,40 @@
+include ../include/Makefile.version
```

Comment:
seeing as this work is already done and will verify that we can separate it out whilst keeping necessary functionality in place and without introducing bugs I would prefer to land this first